File: tapImageApp/app/src/main/java/com/codewithamit/tapimageapp/MainActivity.java
package com.codewithamit.tapimageapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }


    public void fade(View view) {

        ImageView imgview= (ImageView)findViewById(R.id.imageView);
        ImageView imgview1= (ImageView)findViewById(R.id.imageView2);

        imgview.animate().alpha(0f).setDuration(2000);
        imgview1.animate().alpha(1f).setDuration(2000);
    }
}

File: listviewsinkotlin/app/src/main/java/com/example/listviewsinkotlin/MainActivity.kt
package com.example.listviewsinkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.activity_main.view.*
import kotlinx.android.synthetic.main.fruitslistnames.view.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        lvfruits.adapter = ArrayAdapter<String>(
            this,
            R.layout.fruitslistnames,
            R.id.fruitsnames,
            arrayOf("Apple","mango")
        )

        //on click listener
        lvfruits.setOnItemClickListener { parent,view, position, id ->
            Toast.makeText(this,"$position ${view.fruitsnames.text}",Toast.LENGTH_SHORT).show()
        }

    }
}